Jenkinsfile

```diff
@@ -46,7 +47,7 @@ node {
       }
     }
     stage('SelectNamespace') {
-      (kubectlNamespace, lock) = kubeHelper.selectAndLockNamespace(pipeConfig['UID'])
```

Comment:
Sorry just saw this. Andrew made a same change on Jenkinsfile.  https://github.com/uc-cdis/cloud-automation/pull/1068  Post here just in case any conflict shows up.